src/main/java/com/bankofspring/hello/Main.java

```diff
@@ -0,0 +1,28 @@
+package com.bankofspring.hello;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+/**
+ *
+ * @author ram
+ */
+public class Main {
+
+	/**
+	 * @param args for the program
+	 */
+	public static void main(String[] args) {
+		ApplicationContext context = new ClassPathXmlApplicationContext("greeting.xml");
+		
+		GreetingService helloWorld = (GreetingService) context.getBean("helloWorld");
```

Comment:
Trivial: You could consider this method [getBean(String name, Class<T> requiredType)](http://docs.spring.io/spring/docs/3.0.6.RELEASE/api/org/springframework/beans/factory/BeanFactory.html#getBean(java.lang.String, java.lang.Class)) to avoid having to do an explicit cast.